framework/src/main/java/org/checkerframework/framework/stub/RemoveAnnotationsForInference.java

```diff
@@ -65,6 +78,20 @@
  */
 public class RemoveAnnotationsForInference {
 
+  /**
+   * Do not instantiate. This is a standalone program whose entry point is {@link #main(String[])}.
+   */
+  private RemoveAnnotationsForInference() {
+    throw new Error("Do not instantiate RemoveAnnotationsForInference.");
+  }
+
+  /**
+   * A list of annotations not to remove. Used to exclude project-specific annotations that must
+   * remain for the project to build. (It would be burdensome to add all project-specific
+   * annotations to the global list in {@link #isTrustedAnnotation(String)}.)
+   */
+  private static @MonotonicNonNull List<String> excludedAnnotations = null;
```

Comment:
`exclude` and `remove` have similar connotations.  Maybe this should be `nonRemovedAnnotations`.

Reply:
Good point. I'd prefer to use a positive verb, though. I've change it to `keep` throughout, but I'm open to other suggestions if you have a better one.